shell/edit/secret/index.vue

```diff
@@ -237,6 +275,14 @@ export default {
 
       this.$set(this.value, '_type', type);
       this.$emit('set-subtype', this.typeDisplay(type, driver));
+
+      this.secretType = type;
+
+      if (this.mode === _CREATE) {
+        if (type === 'custom') {
```

Comment:
minor thing, but these two if's can be one... `this.mode === _CREATE && type === 'custom'`